generators/client/files-vue.js

```diff
@@ -370,7 +370,7 @@ const vueFiles = {
 };
 
 function writeFiles() {
-    mkdirp(MAIN_SRC_DIR);
+    mkdirp(this.CLIENT_MAIN_SRC_DIR);
```

Comment:
mkdirp isn't required.
```suggestion
```

Reply:
Without modification the folder main/webapp was created =/ 
The writeFiles funtion in file-angular and file-react use `mkdirp(this.CLIENT_MAIN_SRC_DIR);`

Reply:
`mkdirp(this.CLIENT_MAIN_SRC_DIR)` should be removed from every file-*.
We shouldn't do any manipulation on the disc at this point.
The dir will be created by mem-fs when committing the files to the disc.

Reply:
This is unlikely to cause problems, but I prefer to try removing.
Your call.

Reply:
i'm ok to removed mkdirp at this point, this will be done for jhispster v7 ?

Reply:
or i can remove it in this PR i have try without it and it's ok (for .net blueprint)

Reply:
> i'm ok to removed mkdirp at this point, this will be done for jhispster v7 ?

What will be done for jhipster v7? We are on the jhipster v7 branch now.

Reply:
> or i can remove it in this PR i have try without it and it's ok (for .net blueprint)

I prefer that you remove it.

Reply:
done

Reply:
Great.